crates/bws/src/config.rs

```diff
@@ -58,6 +66,34 @@ fn get_config_path(config_file: Option<&Path>, ensure_folder_exists: bool) -> Pa
     config_file
 }
 
+pub(crate) fn get_state_file_path(
```

Comment:
I don't think this is config related, move to a different file?

Reply:
Resolved in [566545d](https://github.com/bitwarden/sdk/pull/388/commits/566545d475abb1fd04eb825090e4227df841f4ed)